src/object/cli_obj.c

```diff
@@ -986,9 +986,10 @@ struct obj_auxi_args {
 	 * request only targets for one shard.
 	 */
 	union {
-		struct shard_rw_args	rw_args;
-		struct shard_punch_args	p_args;
-		struct shard_list_args	l_args;
+		struct shard_rw_args	 rw_args;
+		struct shard_punch_args	 p_args;
+		struct shard_list_args	 l_args;
+		daos_epoch_t		*epoch;
```

Comment:
seems better to define a "struct shard_sync_args" (can just with one "*epoch"), or comment is as for shard sync args here.

Reply:
I will comment.